src/pipx/interpreter.py

```diff
@@ -26,14 +27,22 @@ def has_venv() -> bool:
 # so we try to locate the system Python and use that instead.
 
 
+def find_py_launcher_python(python_version: Optional[str] = None) -> Optional[str]:
+    py = shutil.which("py")
+    if py and python_version:
+        py = subprocess.run(
+            ["py", f"-{python_version}", "-c", "import sys; print(sys.executable)"],
```

Comment:
Could you try using the path returned by `shutil`?

```suggestion
            [py, f"-{python_version}", "-c", "import sys; print(sys.executable)"],
```

Reply:
Hi @chrysle, good catch, I updated it!